config/locales/views/activities/en.yml

```
@@ -67,6 +71,7 @@ en:
         mine: "My activities"
         featured: "Featured activities"
         all: "All activities"
+      draft-notice: "This activity is in draft mode. It is not visible for students."
```

```suggestion
      draft-notice: "This activity is a draft and thus not visible for students."
```

+ het woord "draft" linken naar de docs (zelfde met andere notice strings in deze PR)